modules/porous_flow/test/tests/pressure_pulse/pressure_pulse_1d_2phase_fv.i

```diff
@@ -0,0 +1,245 @@
+  []
+[]
+
+[AuxVariables]
+  [massfrac_ph0_sp0]
+    family = MONOMIAL
+    order = CONSTANT
+    fv = true
+    initial_condition = 1
+  []
+  [massfrac_ph1_sp0]
+    family = MONOMIAL
+    order = CONSTANT
+    fv = true
+    initial_condition = 0
+  []
+[]
+
+[FVKernels]
+  [mass0]
+    type = FVPorousFlowMassTimeDerivative
+    fluid_component = 0
+    variable = ppwater
+  []
+  [flux0]
+    type = FVPorousFlowAdvectiveFlux
+    variable = ppwater
+    gravity = '0 0 0'
+    fluid_component = 0
+  []
+  [mass1]
+    type = FVPorousFlowMassTimeDerivative
+    fluid_component = 1
+    variable = ppgas
+  []
+  [flux1]
+    type = FVPorousFlowAdvectiveFlux
+    variable = ppgas
+    gravity = '0 0 0'
+    fluid_component = 1
+  []
+[]
+
+[UserObjects]
+  [dictator]
+    type = PorousFlowDictator
+    porous_flow_vars = 'ppwater ppgas'
+    number_fluid_phases = 2
+    number_fluid_components = 2
+  []
+  [pc]
+    type = PorousFlowCapillaryPressureVG
+    m = 0.5
+    alpha = 1
+  []
+[]
+
+[FluidProperties]
+  [simple_fluid0]
+    type = SimpleFluidProperties
+    bulk_modulus = 2e9
+    density0 = 1000
+    thermal_expansion = 0
+    viscosity = 1e-3
+  []
+  [simple_fluid1]
+    type = SimpleFluidProperties
+    bulk_modulus = 2e6
+    density0 = 1
+    thermal_expansion = 0
+    viscosity = 1e-5
+  []
+[]
+
+[Materials]
+  [temperature]
+    type = ADPorousFlowTemperature
+  []
+  [ppss]
+    type = ADPorousFlow2PhasePP
+    phase0_porepressure = ppwater
+    phase1_porepressure = ppgas
+    capillary_pressure = pc
+  []
+  [massfrac]
+    type = ADPorousFlowMassFraction
+    mass_fraction_vars = 'massfrac_ph0_sp0 massfrac_ph1_sp0'
+  []
+  [simple_fluid0]
+    type = ADPorousFlowSingleComponentFluid
+    fp = simple_fluid0
+    phase = 0
+  []
+  [simple_fluid1]
+    type = ADPorousFlowSingleComponentFluid
+    fp = simple_fluid1
+    phase = 1
+  []
+  [porosity]
+    type = ADPorousFlowPorosityConst
+    porosity = 0.1
+  []
+  [permeability]
+    type = ADPorousFlowPermeabilityConst
+    permeability = '1E-15 0 0 0 1E-15 0 0 0 1E-15'
+  []
+  [relperm_water]
+    type = ADPorousFlowRelativePermeabilityCorey
+    n = 1
+    phase = 0
+  []
+  [relperm_gas]
+    type = ADPorousFlowRelativePermeabilityCorey
+    n = 1
+    phase = 1
+  []
+[]
+
+[FVBCs]
+  [leftwater]
+    type = FVDirichletBC
+    boundary = left
+    value = 3E6
+    variable = ppwater
+  []
+  [leftgas]
+    type = FVDirichletBC
+    boundary = left
+    value = 3E6
+    variable = ppgas
+  []
+[]
+
+[Preconditioning]
+  [smp]
+    type = SMP
+    full = true
+    petsc_options = '-snes_converged_reason -ksp_diagonal_scale -ksp_diagonal_scale_fix -ksp_gmres_modifiedgramschmidt -snes_linesearch_monitor'
+    petsc_options_iname = '-ksp_type -pc_type -sub_pc_type -sub_pc_factor_shift_type -pc_asm_overlap -snes_atol -snes_rtol -snes_max_it'
+    petsc_options_value = 'gmres      asm      lu           NONZERO                   2               1E-15       1E-20 20'
```

Comment:
are these tight tolerances necessary?

Reply:
Also copy and paste. I'll see if they are really needed

Reply:
Looks like these aren't needed